Week7/Day5/Exercises/TIcTacToe.py

```diff
@@ -8,7 +8,7 @@ def play():
         counter = 1
         gui_list = []
 
-
+        # why you define the function as nested
```

Comment:
I put all the functions and variables outside of the play() function, now it looks prettier. But I struggle with splitting code into functions as I made it really unstable, I mean that it  works in only one certain way and it's really hard to rearrange anything. I really tried)) And next time I'll take it into account